src/platform/packages/private/kbn-grid-layout/grid/grid_layout.tsx

```diff
@@ -98,22 +103,54 @@ export const GridLayout = ({
 
   useEffect(() => {
     /**
-     * This subscription ensures that rows get re-rendered when their orders change
+     * This subscription is responsible for setting both the rendered elements and the
+     * `gridTemplateString` that controls how the grid layout is rendered
```

Comment:
I am wondering if `gridTemplateString` deserves a longer explanation - what it exactly does (you've explained it to me but I'd have to dig deeper if I didn't know) 🤔 nothing urgent though, just sharing my thoughts

Reply:
Good call - added more info in e64365ecb61c0c26baf81a148928ecb3cf15d892